validators/validators.go

```diff
@@ -19,16 +18,18 @@ func ValidateCreateAccountRequest(in *accountspb.CreateAccountRequest) error {
 func ValidateGetAccountRequest(in *accountspb.GetAccountRequest) error {
 	return validation.ValidateStruct(in,
 		validation.Field(&in.Id, validation.When(in.Email == "", validation.Required), is.UUID),
-		validation.Field(&in.Email, validation.By(func(value interface{}) error {
-			return errors.New("get account by email is unimplemented")
-		})),
+		// validation.Field(&in.Email, validation.By(func(value interface{}) error {
+		// 	return errors.New("get account by email is unimplemented")
+		// })),
 	)
 }
 
 func ValidateUpdateAccountRequest(in *accountspb.UpdateAccountRequest) error {
-	return validation.ValidateStruct(in,
-		validation.Field(&in.Account.Id, validation.Required, is.UUID),
+	return validation.Validate(in.Account.Id,
+		validation.Required, // not empty
```

Comment:
Why did you replace `ValidateStruct` with `Validate`?

Reply:
https://github.com/go-ozzo/ozzo-validation/issues/24#issuecomment-566633912

seems like ozzo cannot validate named embed struct